<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Raffle</string>
  </property>
  <widget class="QWidget" name="parentFormLayout">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="addEntryTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Add Entry</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="nameLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="contactLabel">
            <property name="text">
             <string>Email/Phone</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="contactLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="numTicketsLabel">
            <property name="text">
             <string># Tickets</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="numTicketsSpinBox">
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="commentLabel">
            <property name="text">
             <string>Comment</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="commentLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="saveButton">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="viewEntriesTab">
       <attribute name="title">
        <string>View Entries</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="drawWinnerTab">
       <attribute name="title">
        <string>Draw Winner</string>
       </attribute>
       <widget class="QPushButton" name="drawWinnerButton">
        <property name="text">
         <string>Draw Winner</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionOpen_Database">
   <property name="text">
    <string>Open Database</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
